config/console/console_init.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: multicluster-console
+spec:
+  selector:
+    matchLabels:
+      app: odf-multicluster-console
+  template:
+    spec:
+      containers:
+        - name: odf-multicluster-console
+          image: odf-multicluster-console:latest
+          resources:
+            limits:
+              cpu: "100m"
+              memory: "512Mi"
+          ports:
+            - containerPort: 9001
+              protocol: TCP
+          args:
+            - "--ssl --cert=/var/serving-cert/tls.crt --key=/var/serving-cert/tls.key"
+          volumeMounts:
```

Comment:
We can also add a toleration for the ocs nodes (node.ocs.openshift.io/storage) . 
It will scheduling on these nodes as well